python_modules/dagster/dagster_tests/definitions_tests/asset_reconciliation_tests/observable_source_asset_scenarios.py

```diff
@@ -124,4 +129,29 @@
             run_request(asset_keys=["asset1", "asset3"]),
         ],
     ),
+    "unchanging_observable": AssetReconciliationScenario(
+        assets=downstream_of_unchanging_observable_source,
+        unevaluated_runs=[
+            run(["source_asset1"], is_observation=True),
+            run(["asset1"]),
+            run(["source_asset1"], is_observation=True),
```

Comment:
previously, this would result in a new run request, as we weren't looking at the specific versions of each observation.